components/ui/Error.tsx

```diff
@@ -6,10 +6,38 @@ import {Button, Column, Row, Text} from '.';
 import {Header} from './Header';
 import {Theme} from './styleUtils';
 import testIDProps from '../../shared/commonUtil';
+import {Modal} from './Modal';
 
 export const Error: React.FC<ErrorProps> = props => {
```

Comment:
Will we use this same component to show any other errors that might be in the future? If it's specific then set the component title to that particular error.